draft-ietf-quic-tls.md

```diff
@@ -1157,12 +1130,69 @@ key updates in a short time frame succession and significant packet reordering.
 ~~~
 {: #ex-key-update title="Key Update"}
 
+
+## Initiating a Key Update
+
+Endpoints maintain separate read and write secrets for packet protection.  An
+endpoint initiates a key update by updating its packet protection write secret
+and using that to protect new packets.  The endpoint creates a new write secret
+from the existing write secret as performed in Section 7.2 of {{!TLS13}}.  This
+uses the KDF function provided by TLS with a label of "quic ku".  The
+corresponding key and IV are created from that secret as defined in
+{{protection-keys}}.  The header protection key is not updated.
+
+The endpoint uses the key and IV to protect all subsequent packets, and
+increments the value of the Key Phase bits modulo 4 in the short packet header
+to signal the change of keys.
+
+An endpoint MUST NOT initiate more than one key update at a time.  A new key
+cannot be used until the endpoint has received an indication that its peer is
+using the same key phase.
+
+An endpoint that receives a packet protected with old keys that includes an
+acknowledgement for a packet protected with newer keys MAY treat that as a
+connection error of type PROTOCOL_VIOLATION.
+
+
+## Responding to a Key Update
+
+A receiving endpoint detects an update when the Key Phase is one greater than
+what it is expecting.  The endpoint creates a new read secret and the
+corresponding read key and IV using the same process as its peer.
+
+A packet with a Key Phase other than the expected or next value MUST be
```

Comment:
The previous one is ok as well, as long as it’s sent with a smaller packet number.